spec/requests/users_spec.rb

```diff
@@ -1,53 +1,20 @@
 require 'rails_helper'
 
 RSpec.describe "Users", type: :request do
-  describe "GET /index" do
-    it "returns http success" do
-      get users_path
-      expect(response).to have_http_status(:success)
-    end
-  end
-
-  describe "GET /show" do
-    it "returns http success" do
-      get "/users/show"
-      expect(response).to have_http_status(:success)
-    end
-  end
-
-  describe "GET /new" do
-    it "returns http success" do
-      get "/users/new"
-      expect(response).to have_http_status(:success)
-    end
-  end
-
-  describe "GET /create" do
-    it "returns http success" do
-      get "/users/create"
-      expect(response).to have_http_status(:success)
-    end
-  end
+  subject(:new_user) { User.new(name: 'Joao', age: 16, biography: 'A short history') }
 
-  describe "GET /edit" do
-    it "returns http success" do
-      get "/users/edit"
-      expect(response).to have_http_status(:success)
-    end
-  end
-
-  describe "GET /update" do
-    it "returns http success" do
-      get "/users/update"
+  describe 'GET /index' do
+    it 'returns http success' do
+      get users_path
       expect(response).to have_http_status(:success)
     end
   end
 
-  describe "GET /destroy" do
-    it "returns http success" do
-      get "/users/destroy"
+  describe 'GET /show' do
+    it 'returns http success' do
```

Comment:
This example only asserts that the request is successful, but we have no idea whether the action actually does what it's supposed to.